packages/cadl-ranch-specs/http/azure/client-generator-core/client-namespace/client.tsp

```diff
@@ -0,0 +1,40 @@
+import "./main.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using global.Azure.ClientGenerator.Core;
+using _Specs_.Azure.ClientGenerator.Core.ClientNamespace;
+
+@scenario
+@scenarioDoc("""
+  Expected client namespace for clients:
+  - ClientNamespaceFirstClient: Azure.ClientGenerator.Core.ClientNamespace
+  - ClientNamespaceSecondClient: Azure.ClientGenerator.Core.ClientNamespace.Second
+  
+  Expected client namespace for models:
+  - FirstClientResult: Azure.ClientGenerator.Core.ClientNamespace.First
+  - SecondClientResult: Azure.ClientGenerator.Core.ClientNamespace.Second
+  - SecondClientEnumType: Azure.ClientGenerator.Core.ClientNamespace.Second.Sub
+  """)
+namespace Client;
+
+@client({
+  name: "ClientNamespaceFirstClient",
+  service: _Specs_.Azure.ClientGenerator.Core.ClientNamespace,
+})
+@clientNamespace("azure.clientgenerator.core.clientnamespace", "java")
```

Comment:
I assume for other languages:

csharp: `Azure.ClientGenerator.Core.ClientNamespace`
python: `azure.client_generator.core.client_namespace`

Let me know what the pattern for Go/JS. I will fill in here before PR public for review.